source/javascripts/stats/main.js.erb

```diff
@@ -4,11 +4,11 @@ function init() {
 	addCustomTablesorterParser();
 	loadFilterList();
 
-	newStatsTab("allocators", "Registrations by Allocators", initMainStats("allocator_facet"), "datacentres", "Allocator");
-	newStatsTab("datacentres", "Registrations by Datacentres", initMainStats("datacentre_facet"), "prefixes", "Datacentre");
-	newStatsTab("prefixes", "Registrations by Prefixes", initMainStats("prefix"), "resolution-report", "Prefixes", {headers: { 0: { sorter:'prefixes' }}});
+	newStatsTab("allocators", "", initMainStats("allocator_facet"), "datacentres", "Allocator");
```

Comment:
I know we're still working out language but should we still keep the old naming?

Reply:
now its provider and clients